interfaces/builtin/network_setup_control.go

```diff
@@ -19,17 +19,31 @@
 
 package builtin
 
+import "github.com/snapcore/snapd/release"
+
 const networkSetupControlConnectedPlugAppArmor = `
 # Description: Can read/write netplan configuration files
 
 /etc/netplan/{,**} rw,
 /etc/network/{,**} rw,
 `
 
+const networkSetupControlConnectedPlugAppArmorClassic = `
+# Description: Can read/write NetworkManager's dispatcher (nm-dispatcher) scripts
+
+/etc/NetworkManager/dispatcher.d/{,**} rw,
```

Comment:
This should be changed to:
```
# On classic, allow snaps to manage nm-dispatcher scripts
/etc/NetworkManager/dispatcher.d/{,**} r,
/etc/NetworkManager/dispatcher.d/** w,
```

This makes the comment consistent with how we handle individual rules elsewhere and more importantly it makes it so that the /etc/NetworkManager/dispatcher.d is not writable for the snap.